docs/use_cases/prometheus-alerts-with-kubescape.md

```diff
@@ -0,0 +1,49 @@
+In the complex environment of a cloud-native enterprise, ensuring the security and stability of Kubernetes clusters is crucial. The following narrative illustrates how a company leverages Kubescape and Prometheus to establish a robust monitoring and alerting framework for their Kubernetes clusters.
+
+
+---
+
+
+#### **Scenario**
+
+An operations team at a leading cloud services provider is focused on bolstering the observability of their Kubernetes clusters. With a variety of services running across different environments, the team prioritizes the setup of Prometheus for metrics collection and alerts for security incidents as identified by Kubescape, allowing for quick and effective incident response.
+
+
+## **Actions**
+
+
+
+1. Prometheus Installation:
+    * The team installs Prometheus using Helm charts, setting up the server, Alertmanager, and node exporters to gather cluster metrics.
+    * They ensure Prometheus has the necessary RBAC permissions for metrics scraping from cluster nodes and pods.
+2. Kubescape Metrics Integration:
+    * Prometheus is then configured to scrape security metrics exposed by Kubescape.
+    * In the absence of native Prometheus-compatible metrics from Kubescape, an intermediary service or exporter is deployed to format Kubescape's output for Prometheus.
```

Comment:
Do we have an example or suggestion for this intermediary? I think we should.

Reply:
Decided to remove this bullet